mdpow/tests/test_analysis.py

```diff
@@ -4,17 +4,18 @@
 
 import yaml
 import pybol
-
+import gromacs
 from numpy.testing import assert_array_almost_equal
 
 from six.moves import cPickle as pickle
 
-import mdpow.fep
```

Comment:
leave as before with full paths as it makes it MUCH clearer where code comes from

Reply:
I'll make that replacement in fep and equil in a new commit.